engine/platform/linux/vid_fbdev.c

```diff
@@ -9,7 +9,9 @@
 
 #include <linux/fb.h>
 #include <sys/mman.h>
+#if !XASH_PSVITA
```

Comment:
Wait, how does this file get enabled on PSVita?

Reply:
It doesn't, I probably blanket-replaced `#include <sys/ioctl.h>` or something. Removed.